Changes

```diff
@@ -264,10 +264,13 @@ OCaml 5.0
   Guillaume Munch-Maccagnoni, additional discussions with Stephen
   Dolan and Luc Maranget)
 
-- #5299, #4787, #11138, #11272: `Caml_state` is `NULL` when the domain lock is
-  not held. This allows to test if the current thread holds the lock of its
-  domain, and it makes it simpler to debug when one forgets to acquire
-  the lock.
+- #5299, #4787, #11138, #11272, #?: To help debugging, `Caml_state`
```

Comment:
The `?` in the  PR number should be updated.

Reply:
```suggestion
- #5299, #4787, #11138, #11272, #11506: To help debugging, `Caml_state`
```